ravenpy/models/rv.py

```diff
@@ -881,7 +898,15 @@ def _parser(lines, indent="", fmt=str):
         if header:
             new_indent, key, value = header.groups()
             if new_indent > indent:
-                out[old_key] = _parser(itertools.chain([line,], lines), new_indent)
+                out[old_key] = _parser(
+                    itertools.chain(
+                        [
+                            line,
+                        ],
+                        lines,
+                    ),
+                    new_indent,
+                )
```

Comment:
black seems to be aggressive on this one.

Reply:
I fixed it. Note that the reason for this extremism was the pre-existing trailing comma for this list: `[line,]`

> A pre-existing trailing comma informs Black to always explode contents of the current bracket pair into one item per line. Read more about this in the Pragmatism section below.